### PR TITLE
Add support for python3.6

### PR DESCRIPTION
**Pull request by pokoli**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,16 +3,27 @@ language: python
...
+     - python: 2.7
+       env: TOXENV=coverage,codecov
+     - python: 3.4
+       env: TOXENV=py34
+     - python: 3.5
+       env: TOXENV=py35
+     - python: 3.5
+       env: TOXENV=py35-functional
+     - python: 3.6
+       env: TOXENV=py36
```
Comment (dims) [inline .travis.yml]: Can you please add a py36-functional as well?

Comment (pokoli) [inline .travis.yml]: Done

**Comment by mbohlool** [comment]
Looks good. Can you stash commits into one?

**Comment by dims** [comment]
LGTM 👍 (+1 to Squash)

**Comment by pokoli** [comment]
Squashed all the commits into single one.

**Pull request merged**